server/src/dto/app-authentication.dto.ts

```diff
@@ -29,6 +29,7 @@ export class AppForgotPasswordDto {
 
 export class AppPasswordResetDto {
   @IsString()
+  @Transform(({ value }) => value.trim())
```

Comment:
```suggestion
  @Transform(({ value }) => value?.trim())
```
to avoid error if value is empty